pkg/model/combiner.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+import (
+	"github.com/go-kit/kit/log/level"
+
+	"github.com/cortexproject/cortex/pkg/util/log"
+)
+
+type objectCombiner struct{}
+
+var ObjectCombiner = objectCombiner{}
```

Comment:
Any particular reason to have the struct as an exported variable?

Reply:
There's no state so it seems unnecessary to reinstantiate it constantly.  Seemed it easier to do it once and use it whereever.